src/DataTable/RowsPerPageInput/styles.scss

```diff
@@ -0,0 +1,23 @@
+// please don't focus on my design, I just want to demo BEM methodology
```

Comment:
as I said in comment, I think BEM is a nice way to organise css.
here is the [introduction link]( http://getbem.com/introduction/ )